scripts/build.sh

```
@@ -0,0 +1,102 @@
+# List generated themes
+info "Generated themes:"
+for theme in themes/*.json; do
+    echo "  - $(basename "$theme")"
+done
+
+# Step 5: Package extension (optional)
+if [ "$1" == "--package" ]; then
+    info "Creating VSIX package..."
+    vsce package
+    
+    VSIX_FILE=$(ls *.vsix 2>/dev/null | head -n 1)
```

Shellcheck (0.10.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)